.github/workflows/publish.yml

```diff
@@ -2,60 +2,56 @@ name: Publish
 
 on:
   push:
-    branches: ["main"]
-    tags: ["*.*.*"]
-    paths-ignore: ["*.md"]
+    branches:
+    - main
 
-env:
```

Comment:
See https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/